src/AbstractFixer.php

```diff
@@ -97,6 +97,9 @@ public function getName()
 
     /**
      * {@inheritdoc}
+     *
+     * Must run before PhpdocAlignFixer.
+     * Must run after CommentToPhpdocFixer, PhpdocIndentFixer, PhpdocScalarFixer, PhpdocToCommentFixer, PhpdocTypesFixer.
```

Comment:
I don't think this is true for every subclass of the AbstractFixer, at least not in design, I think it is better to document on the child classes only.

Reply:
Fixed. Not sure why I added the comment here...